### PR TITLE
feat: Like post

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- like post
- remove like
- fetch total number of likes

### CODE REVIEW

File: src/pages/bookmark/bookmarkContent.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export const BookMarkContent = () => {
+  const { bookmarkList } = useAppSelector((store) => store.user);
+  const { allPosts } = useAppSelector((store) => store.posts);
+  const {allUsers} = useAppSelector(store => store.auth)
+  const bookmarkData = useBookmark()
+  const dispatch = useAppDispatch()
+
+  // useEffect(() => {
+  //   dispatch(getAllUsers())
```
Comment (vermasavita) [inline src/pages/bookmark/bookmarkContent.tsx]: you can remove these comments

File: src/pages/bookmark/bookmarkContent.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      ) : (
+        bookmarkList?.map((item, index) => <PostCard item={item} key={index} />)
+      )}
+    </div>
+  );
+};
+
+// content
+// "Shree Krishna Govinda Hare Murari Hae Nath Narayan Vasudeva Radhae, Radhae!!"
+// id
```
Comment (vermasavita) [inline src/pages/bookmark/bookmarkContent.tsx]: you can remove these comments

File: src/pages/bookmark/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                <div className="relative xs:col-span-8 md:col-span-6 lg:col-span-4 border-l-2 border-r-2">
+                  <BookMarkContent />
+                </div>
+                <div className="hidden lg:block col-span-2">
+                  <UserSidebar />
+                </div>
+          
+                {/* TODO: For future reference */}
+                {/* <div className="col-span-8 p-3 bg-green-100">
+                  <Footer />
```
Comment (vermasavita) [inline src/pages/bookmark/index.tsx:25]: you can remove these comments

File: src/pages/home/mainContent.tsx [hunk 4]
```diff
@@ -5,13 +5,25 @@ import { EditPostModal } from "components/common/modal/EditPostModal";
...
+
+  const { bookmarkList } = useAppSelector((store) => store.user);
+  const bookmarkData = useBookmark()
+  
   useEffect(() => {
     dispatch(getAllPosts());
+    dispatch(getBookmarkList(bookmarkData))
   }, [allPosts]);
+
+  // console.log(bookmarkList)
```
Comment (vermasavita) [inline src/pages/home/mainContent.tsx]: remove console

**Review by vermasavita — COMMENTED** [review]
Functionalities are working fine. Just remove the commented code.

**Review by vermasavita — APPROVED** [review]

**Review by Digambar1411 — APPROVED** [review]
All functionalities are working fine. can be merged.

**Pull request merged**